pkg/cmd/repo/fork/fork_test.go

```diff
@@ -409,23 +441,35 @@ func TestRepoFork_in_parent(t *testing.T) {
 
 func TestRepoFork_outside(t *testing.T) {
 	tests := []struct {
-		name string
-		args string
+		name  string
+		args  string
+		toOrg bool
 	}{
 		{
-			name: "url arg",
-			args: "--clone=false http://github.com/OWNER/REPO.git",
+			name:  "url arg",
+			args:  "--clone=false http://github.com/OWNER/REPO.git",
+			toOrg: false,
 		},
 		{
-			name: "full name arg",
-			args: "--clone=false OWNER/REPO",
+			name:  "full name arg",
+			args:  "--clone=false OWNER/REPO",
+			toOrg: false,
+		},
+		{
+			name:  "fork to org with clone",
+			args:  "--clone=false OWNER/REPO --org batmanshome",
+			toOrg: true,
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			defer stubSince(2 * time.Second)()
 			reg := &httpmock.Registry{}
-			defer reg.StubWithFixturePath(200, "./forkResult.json")()
+			if tt.toOrg {
+				defer reg.StubWithFixturePath(200, "./orgForkResult.json")()
```

Comment:
Instead of switching stubs based on a boolean in the test case, consider letting the test case define its own stub.